src/ingestion.py

```diff
@@ -0,0 +1,24 @@
+from gremlin_connect.gremlin_adapter import GremlinAdapter
+
+from src.graph_model import Traversel
+from src.ingestion_data import IngestionData
+
+gremlin_adapter = GremlinAdapter()
+def _ingest_pcve(pcve):
+    g = Traversel()
```

Comment:
This looks to me that you may want to use gremlin websocket. If my assumption is correct, can you please help me understand the benefits over http?

Reply:
It is a [recommendation from Gremlin community](http://tinkerpop.apache.org/docs/current/reference/#_connecting_via_http)[1] to use bytecode based implementation through WebSocket. 

But current implementation is still based on HTTP.

[1] http://tinkerpop.apache.org/docs/current/reference/#_connecting_via_http